dev/tests/integration/framework/Magento/TestFramework/Annotation/ConfigFixture.php

```diff
@@ -46,7 +47,7 @@ class ConfigFixture
      * @param string|bool|null $scopeCode
      * @return string
      */
-    protected function _getConfigValue($configPath, $scopeCode = null)
+    protected function _getConfigValue($configPath, $scopeCode = null, $scopeType = ScopeInterface::SCOPE_STORE)
```

Comment:
Change parameter type in protected methods is not allowed see our
[**Backward Compatible Guide**](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)